extensions/ima/src/main/java/com/google/android/exoplayer2/ext/ima/ImaAdsLoader.java

```diff
@@ -159,6 +176,19 @@ public Builder setMediaLoadTimeoutMs(int mediaLoadTimeoutMs) {
       return this;
     }
 
+    /**
+     * Sets the ad media maximum recommended bitrate, in Kbps.
+     *
+     * @param mediaBitrateKbps The ad media maximum recommended bitrate, in Kbps.
+     * @return This builder, for convenience.
+     * @see AdsRenderingSettings#setBitrateKbps(int)
+     */
+    public Builder setMediaBitrateKbps(int mediaBitrateKbps) {
```

Comment:
nit - From a consistency point of view with respect to other ExoPlayer APIs, it's probably better if this is `setMaxMediaBitrate(int bitrate)` with bitrate in bits per second. The conversion to kbps can happen when it's actually passed to IMA in the `adsRenderingSettings` call.

Reply:
Thank you for your review.
You mean that you should divide bps by 1000?

```
public Builder setMaxMediaBitrate(int bitrate) {
  this.bitrate = bitate
}
...
private void startAdPlayback() {
  ...
  if (bitrate != BITRATE_UNSET) {
    adsRenderingSettings.setBitrateKbps(bitrate / 1000);
  }
  ...
}
```

The remainder is truncated. Is this okay?

```
adsLoader = new ImaAdsLoader.Builder(context)
    .setMaxMediaBitrate(1500xxx)
    .buildForAdTag(Uri.parse(adTag));
```

Reply:
Yes, exactly that!